tests/data/vpaid.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
vast.xml is in tests/ dir. move it to same dir too?

Reply:
Last time I did tests I was unhappy with how it written. XML is unreadable within php. But at that time I had no beautiful enough and working solution. Now I have - and after we close this PR I'm planning to send one more - with all XMLs extracted to files from php. And, maybe, I will add one more test - create `tests/data/iab/` where I'll put all IAB sample vasts and recreate them using lib. Putting all vasts inside `tests/` will create a mess for sure.
But this refactoring should be separated from current task - adding VPAID support.